### DISCUSSION TITLE
Is the implement missing item a feature in helix?

### DISCUSSION THREAD

**Discussion by norman784** [helix-editor/helix#2009] [Q&A] [answered]
rust-analyzer has a feature to implement missing items for a trait, do that exists in helix?

![image](https://user-images.githubusercontent.com/204656/162264747-c1b8e872-7b13-42ef-891b-e7cf67c3e2de.png)

This prompt in vscode

![image](https://user-images.githubusercontent.com/204656/162264809-94deda80-bf6f-4692-9316-cad942da4a1c.png)

**Comment by CBenoit** [comment, marked as answer]
Yes, it's a code action you can trigger with `space`-`a` when main cursor is at an appropriate location (somewhere in the impl block).
![image](https://user-images.githubusercontent.com/3809077/162267388-d11e0f23-f2a0-4270-a338-30f2388485dc.png)
![image](https://user-images.githubusercontent.com/3809077/162267432-11b124f6-bc39-4af9-827a-2975159e7be2.png)
↓
![image](https://user-images.githubusercontent.com/3809077/162267507-556bb5b5-e154-47d8-ae47-d010815f3bc1.png)

**Reply by norman784** [reply]
That's why I didn't found it before, the name of this feature is helix is called `Perform code action`.

Thanks
